public/script/workTrolly.js

```diff
@@ -0,0 +1,15 @@
+$("#createNewTask").ready(function() {
```

Comment:
Please update the file name as requested

Reply:
okay